src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -556,6 +557,13 @@ public Boolean apply(WebDriver driver) {
               return driver.findElements(
                   By.cssSelector(loggedInUserSelectorMonobook.replace("%userName%",
                       userName.replace(" ", "_")))).size() > 0;// only for verification
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)